### DISCUSSION TITLE
Welcome to cordova-plugin-googlemaps Discussions!

### DISCUSSION THREAD

**Discussion by wf9a5m75** [mapsplugin/cordova-plugin-googlemaps#2846] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Mohammad-Toufiq-Pathan** [comment]
Hi getting plugin.google.maps.PluginEnvironment.isAvailable is Nosuchmethod error for release build, working fine debug build in Android!
<img src="https://user-images.githubusercontent.com/48539642/106142213-390c5e80-6197-11eb-86ff-cea0b26de8c5.jpg" width="300">

**Reply by wf9a5m75** [reply]
I have no idea. Please share your project files on GitHub repository, and share it with me

**Reply by wf9a5m75** [reply]
This plugin does not support "proguard". Please disable it

**Reply by nosTa1337** [reply]
@wf9a5m75 what about asking some people if they join mainting this plugin? It seems you are busy. As a developer myself I can understand that but I bet you can find people willing to spend their time mainting this plugin :)

**Comment by taxman007** [comment]
Any chance there will be an update to this plugin for compatibility with Ionic 7? 

Thanks!

**Reply by halie412** [reply]
Do You get error with ionic 7?

**Reply by taxman007** [reply]
Yes, I had a capability issue because of "@ionic-native/google-maps": "^5.27.0-beta-20200630"

**Comment by outofcontrolmike** [comment]
Hello!

We are getting the following message below from Google Maps Platform in regards to iOS version 15 being no longer supported in May 2025.

I'm curious if this plugin will handle the scenario of still supporting iOS 15 or if it will take some custom work on other dev's ends to keep version 15 compatible with this plugin.

Please let me know if this is not the correct place to provide this for coming issue, thank you.

---

<html>
<body>
<p data-sourcepos="7:1-7:27">Hello Google Maps Customer,</p>
<p data-sourcepos="9:1-9:433">We are writing to remind you about an upcoming update to our <a data-sourcepos="9:62-9:425" href="https://link.edgepilot.com/s/b3a91a5d/q1YGlIV_4EeDR8HAeE2r8w?u=https://notifications.google.com/g/p/ANiao5pex2ojm2Jm0FbN-_Q9pSxLNiNMQ7lzsTMIbLl-t-acne1lZYvJz4HYwzQ8cOn6uBZen_rL6htKPEF6ATeuewEXnDUReGog9b6xV_nEXfLykoc5FjjsFDV1pnuMeJ5_bXpCUDz1f3ElIqHofKohR1WyatAasJCxtgB-B9sSpBXt1Fl3sT8DdGiS15BDZHblgi0-EafrVKXisPUwgbCJD5fbO7vHO9ZA6IOS7A" rel="nofollow noreferrer noopener" target="_blank">Mobile OS version support</a> policy.</p>
<p data-sourcepos="11:1-11:143">Starting <strong data-sourcepos="11:10-11:21">May 2025</strong>, a significant update to our <strong data-sourcepos="11:52-11:88">Maps, Places, and Navigation SDKs</strong> for iOS will no longer support devices running iOS 15.</p>
<p data-sourcepos="13:1-13:79">We’ve provided additional information below to guide you through this change.</p>
<h3 data-sourcepos="15:1-15:28">
<a href="#what-do-you-need-to-know" aria-hidden="true" class="anchor" id="user-content-what-do-you-need-to-know"></a>What do you need to know</h3>
<p data-sourcepos="17:1-17:183">Starting <strong data-sourcepos="17:10-17:21">May 2025</strong>, all newly released versions of Maps, Places, and Navigation for iOS will only support iOS 16 or later. The earlier SDK versions will continue supporting iOS 15.</p>
<p data-sourcepos="19:1-19:184">If your dependencies do not specify a version number, your Integrated Development Environment (IDE) will load the newest SDK version and new builds of your app will not support iOS 15.</p>
<h3 data-sourcepos="21:1-21:26">
<a href="#what-do-you-need-to-do" aria-hidden="true" class="anchor" id="user-content-what-do-you-need-to-do"></a>What do you need to do</h3>
<p data-sourcepos="23:1-23:541">You are required to <a data-sourcepos="23:21-23:368" href="https://link.edgepilot.com/s/b2074307/ygUD667UdU25DhrBqF5i_g?u=https://notifications.google.com/g/p/ANiao5qcN5MiJzx_3jUK7BL0llEf5xHoi6kKaTNnKYAKBDZvLTwbt2qo5gTTt4fDYo8jagBjFkDziBTWqylbfHAwkeHsudbQAv1YFCskIAagSdIaqSnS9MeOjpfG1HuoHWZVT9FTDgb3E0k4Q6079uRYdrI12JI0VxDKLm1dCD0_iDNc2Jn8765tEklp8JAeAgV0irr7dOd6DIfjVPGmJ1KEr97vXHYHBjA" rel="nofollow noreferrer noopener" target="_blank">specify a version</a> of the Maps, Places, and Navigation SDKs for iOS in your application's build dependencies to control when you raise the minimum supported OS for newer versions of your app.</p>
<p data-sourcepos="25:1-25:195">Our records show that the projects listed in the table below are used on devices running iOS 15. To ensure continued support, update your SDK dependencies in the next release before <strong data-sourcepos="25:183-25:194">May 2025</strong>.</p>

Project name (project number) | Package name | Impacted SDK | Usage percentage for devices running on iOS 15 or before.
-- | -- | -- | --
My Project (986359021638) | com.farmwaycu.Farmway | Maps SDK | 5


<h3 data-sourcepos="66:1-66:24">
<a href="#were-here-to-help" aria-hidden="true" class="anchor" id="user-content-were-here-to-help"></a>We’re here to help</h3>
<p data-sourcepos="68:1-68:527">We understand that making this change may require some planning but we are here to help. If you have any questions or require any further assistance, please contact your <a data-sourcepos="68:171-68:526" href="https://link.edgepilot.com/s/af28ad99/MWNugZTH1kmwpBXWWeeSFQ?u=https://notifications.google.com/g/p/ANiao5rS5Qh76E7NO1dwrD5uy0KNmT70KDoGbJv4KMdLVmT3eAJSK7OT2g4eE51nfYAWbw_tmSgT2vA4V3ASpr3U1RsizdXKg3KMCIGlq1YIUe2d2C2zMYnrY61bDv97ALlPwkmmuRnOrFdwxJWsZwDjAVHpg_IKDn7YWWuaj-e8L8jndIhfwKez8tOBBYa4IylWtQXoCi-L37XBkP3yKuyJH6RBkCB3" rel="nofollow noreferrer noopener" target="_blank">Google Maps Platform Support</a>.</p>
<p data-sourcepos="70:1-70:41">Thanks for choosing Google Maps Platform.</p>
</body>
</html>Hello Google Maps Customer,

We are writing to remind you about an upcoming update to our [Mobile OS version support](https://link.edgepilot.com/s/b3a91a5d/q1YGlIV_4EeDR8HAeE2r8w?u=https://notifications.google.com/g/p/ANiao5pex2ojm2Jm0FbN-\_Q9pSxLNiNMQ7lzsTMIbLl-t-acne1lZYvJz4HYwzQ8cOn6uBZen_rL6htKPEF6ATeuewEXnDUReGog9b6xV_nEXfLykoc5FjjsFDV1pnuMeJ5_bXpCUDz1f3ElIqHofKohR1WyatAasJCxtgB-B9sSpBXt1Fl3sT8DdGiS15BDZHblgi0-EafrVKXisPUwgbCJD5fbO7vHO9ZA6IOS7A) policy.

Starting **May 2025**, a significant update to our **Maps, Places, and Navigation SDKs** for iOS will no longer support devices running iOS 15.

We’ve provided additional information below to guide you through this change.

### What do you need to know

Starting **May 2025**, all newly released versions of Maps, Places, and Navigation for iOS will only support iOS 16 or later. The earlier SDK versions will continue supporting iOS 15.

If your dependencies do not specify a version number, your Integrated Development Environment (IDE) will load the newest SDK version and new builds of your app will not support iOS 15.

### What do you need to do

You are required to [specify a version](https://link.edgepilot.com/s/b2074307/ygUD667UdU25DhrBqF5i_g?u=https://notifications.google.com/g/p/ANiao5qcN5MiJzx_3jUK7BL0llEf5xHoi6kKaTNnKYAKBDZvLTwbt2qo5gTTt4fDYo8jagBjFkDziBTWqylbfHAwkeHsudbQAv1YFCskIAagSdIaqSnS9MeOjpfG1HuoHWZVT9FTDgb3E0k4Q6079uRYdrI12JI0VxDKLm1dCD0_iDNc2Jn8765tEklp8JAeAgV0irr7dOd6DIfjVPGmJ1KEr97vXHYHBjA) of the Maps, Places, and Navigation SDKs for iOS in your application's build dependencies to control when you raise the minimum supported OS for newer versions of your app.

Our records show that the projects listed in the table below are used on devices running iOS 15. To ensure continued support, update your SDK dependencies in the next release before **May 2025**.

<table>
<tr>
<td>

**Project name (project number)**
</td>
<td>

**Package name**
</td>
<td>

**Impacted SDK**
</td>
<td>

**Usage percentage for devices running on iOS 15 or before.**
</td>
</tr>
<tr>
<td>

[My Project (986359021638)](https://link.edgepilot.com/s/a608e202/4qPLwluDY0a3DgjJF1FH7A?u=https://notifications.google.com/g/p/ANiao5qOyaqspiFVV609UBOu-zE3uanMROUkQW_5b8h2M5ed6l_PEJP01rRCrJivC5EvUhz3N5MedKi1U4ZSqdJoqPYXuH3k_jxYs3CwMQ33KGpiE1sAB9e1ig-7k1yPa7BwseYGELDAMkYc8cjt1EowwxiVhbLvGLbxlen-EziRmbTSX9ac3N_IHv27WbJ04IUXhzQhrMNK29An03M)
</td>
<td>

* com.farmwaycu.Farmway
</td>
<td>

* [Maps SDK](https://link.edgepilot.com/s/f3d84f8e/Kw4LV4-mKkOTvSHIm-ySlA?u=https://notifications.google.com/g/p/ANiao5o44xlgYwWFZooYUg7Xc9sSg47HUxDtEqEYKOw_FYY7Pjp74yo6f39q8p2kGBVD3EJxLjSAgDO71XlDmxr9W0OeRwX-DDyBtnHvkm1U844LzCFUDKnDydjGRkqpZX8F7p5UL8eLKXOVIa86hmX9dPbLvMXWkk1\_-kgbstSCaER3S14qm5hI_2yVh-VQ6Ozh4O2hm3yGmhztix__7DFjoi-\_CUroNLc)
</td>
<td>

* 5
</td>
</tr>
</table>

### We’re here to help

We understand that making this change may require some planning but we are here to help. If you have any questions or require any further assistance, please contact your [Google Maps Platform Support](https://link.edgepilot.com/s/af28ad99/MWNugZTH1kmwpBXWWeeSFQ?u=https://notifications.google.com/g/p/ANiao5rS5Qh76E7NO1dwrD5uy0KNmT70KDoGbJv4KMdLVmT3eAJSK7OT2g4eE51nfYAWbw_tmSgT2vA4V3ASpr3U1RsizdXKg3KMCIGlq1YIUe2d2C2zMYnrY61bDv97ALlPwkmmuRnOrFdwxJWsZwDjAVHpg_IKDn7YWWuaj-e8L8jndIhfwKez8tOBBYa4IylWtQXoCi-L37XBkP3yKuyJH6RBkCB3).

Thanks for choosing Google Maps Platform.Hello Google Maps Customer,
We are writing to remind you about an upcoming update to our [Mobile OS version support](https://link.edgepilot.com/s/b3a91a5d/q1YGlIV_4EeDR8HAeE2r8w?u=https://notifications.google.com/g/p/ANiao5pex2ojm2Jm0FbN-_Q9pSxLNiNMQ7lzsTMIbLl-t-acne1lZYvJz4HYwzQ8cOn6uBZen_rL6htKPEF6ATeuewEXnDUReGog9b6xV_nEXfLykoc5FjjsFDV1pnuMeJ5_bXpCUDz1f3ElIqHofKohR1WyatAasJCxtgB-B9sSpBXt1Fl3sT8DdGiS15BDZHblgi0-EafrVKXisPUwgbCJD5fbO7vHO9ZA6IOS7A) policy.
Starting May 2025, a significant update to our Maps, Places, and Navigation SDKs for iOS will no longer support devices running iOS 15.
We’ve provided additional information below to guide you through this change.

What do you need to know
Starting May 2025, all newly released versions of Maps, Places, and Navigation for iOS will only support iOS 16 or later. The earlier SDK versions will continue supporting iOS 15.
If your dependencies do not specify a version number, your Integrated Development Environment (IDE) will load the newest SDK version and new builds of your app will not support iOS 15.

What do you need to do
You are required to [specify a version](https://link.edgepilot.com/s/b2074307/ygUD667UdU25DhrBqF5i_g?u=https://notifications.google.com/g/p/ANiao5qcN5MiJzx_3jUK7BL0llEf5xHoi6kKaTNnKYAKBDZvLTwbt2qo5gTTt4fDYo8jagBjFkDziBTWqylbfHAwkeHsudbQAv1YFCskIAagSdIaqSnS9MeOjpfG1HuoHWZVT9FTDgb3E0k4Q6079uRYdrI12JI0VxDKLm1dCD0_iDNc2Jn8765tEklp8JAeAgV0irr7dOd6DIfjVPGmJ1KEr97vXHYHBjA) of the Maps, Places, and Navigation SDKs for iOS in your application's build dependencies to control when you raise the minimum supported OS for newer versions of your app.
Our records show that the projects listed in the table below are used on devices running iOS 15. To ensure continued support, update your SDK dependencies in the next release before May 2025.



Project name (project number)


Package name


Impacted SDK


Usage percentage for devices running on iOS 15 or before.




[My Project (986359021638)](https://link.edgepilot.com/s/a608e202/4qPLwluDY0a3DgjJF1FH7A?u=https://notifications.google.com/g/p/ANiao5qOyaqspiFVV609UBOu-zE3uanMROUkQW_5b8h2M5ed6l_PEJP01rRCrJivC5EvUhz3N5MedKi1U4ZSqdJoqPYXuH3k_jxYs3CwMQ33KGpiE1sAB9e1ig-7k1yPa7BwseYGELDAMkYc8cjt1EowwxiVhbLvGLbxlen-EziRmbTSX9ac3N_IHv27WbJ04IUXhzQhrMNK29An03M)



com.farmwaycu.Farmway




[Maps SDK](https://link.edgepilot.com/s/f3d84f8e/Kw4LV4-mKkOTvSHIm-ySlA?u=https://notifications.google.com/g/p/ANiao5o44xlgYwWFZooYUg7Xc9sSg47HUxDtEqEYKOw_FYY7Pjp74yo6f39q8p2kGBVD3EJxLjSAgDO71XlDmxr9W0OeRwX-DDyBtnHvkm1U844LzCFUDKnDydjGRkqpZX8F7p5UL8eLKXOVIa86hmX9dPbLvMXWkk1_-kgbstSCaER3S14qm5hI_2yVh-VQ6Ozh4O2hm3yGmhztix__7DFjoi-_CUroNLc)




5





We’re here to help
We understand that making this change may require some planning but we are here to help. If you have any questions or require any further assistance, please contact your [Google Maps Platform Support](https://link.edgepilot.com/s/af28ad99/MWNugZTH1kmwpBXWWeeSFQ?u=https://notifications.google.com/g/p/ANiao5rS5Qh76E7NO1dwrD5uy0KNmT70KDoGbJv4KMdLVmT3eAJSK7OT2g4eE51nfYAWbw_tmSgT2vA4V3ASpr3U1RsizdXKg3KMCIGlq1YIUe2d2C2zMYnrY61bDv97ALlPwkmmuRnOrFdwxJWsZwDjAVHpg_IKDn7YWWuaj-e8L8jndIhfwKez8tOBBYa4IylWtQXoCi-L37XBkP3yKuyJH6RBkCB3).
Thanks for choosing Google Maps Platform.
